### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by chaeya**
First of all, I think this project is a great open source project.

Today I found that there is no Korean translation and added a Korean translation based on the latest translation template pot.
Please review the PR.

### CODE REVIEW

File: po/ko.po [hunk 1]
```diff
@@ -0,0 +1,2154 @@
...
+
+#: src/main.py
+msgid "Report bugs or ideas"
+msgstr "버그 또는 아이디어보고"
+
+#. To tranlators: "translate" this by a list of your names (one name
+#. per line), they will be displayed in the "about" dialog
+#: src/main.py
+msgid "translator-credits"
+msgstr "번역자 크레딧"
```
Comment (maoschanz) [inline po/ko.po:843]: see the comment above the label

File: po/ko.po [hunk 2]
```diff
@@ -0,0 +1,2154 @@
...
+msgid "Automatic"
+msgstr "자동"
+
+#: src/preferences.py
+msgid "Compact"
+msgstr "콤팩트"
+
+#: src/preferences.py
+msgid "elementary OS"
+msgstr "elementray 스타일"
```
Comment (maoschanz) [inline po/ko.po:1096]: elementary

File: po/ko.po [hunk 3]
```diff
@@ -0,0 +1,2154 @@
...
+msgstr "그림판"
+
+#: data/com.github.maoschanz.drawing.desktop.in
+msgid "Simple drawing utility"
+msgstr "간단한 그리기 유틸리티"
+
+#. This is a list of keywords. "Paint" should not be translated, the others should. Don't forget the semicolons.
+#: data/com.github.maoschanz.drawing.desktop.in
+msgid "Paint;Sketch;Pencil;"
+msgstr "페인트, 스케치, 연필;"
```
Comment (maoschanz) [inline po/ko.po:33]: this should be "Paint;페인트;스케치;연필;"

**Review by maoschanz — COMMENTED** [review]
Hello, thank you a lot for this translation, i've made a few comments

Concerning the underscores to activate the menu bar from the keyboard, i noticed an inconsistency but i'm not sure whether or not you should change it: you sometimes put an underscore before an hangeul, and sometimes you add a latin letter. Is this normal? Sorry i don't know how korean keyboards work

**Comment by maoschanz** [comment]
You can add commits to your version of the repository, and they will be added here automatically until i merge them, so there is no need to open a new "pull request"

File: po/ko.po (hunk 1 above)
Comment (chaeya) [reply po/ko.po]: Removed "translator-credits" msgstr

File: po/ko.po (hunk 2 above)
Comment (chaeya) [reply po/ko.po]: Fixed "elementary OS" spell

File: po/ko.po (hunk 3 above)
Comment (chaeya) [reply po/ko.po]: Changed to "Paint;그림판;페인트;스케치;"

**Comment by chaeya** [comment]
First of all, thanks for the advice.
1) I have modified the code to read following the comments you reviewed.
2) And the shortcuts on the top menu have consistently changed the text. As a result of the test, it works normally in the Korean environment.

Please review it again.

**Comment by maoschanz** [comment]
Thank you very much!

**Pull request merged**